test/parallel/test-http-client-headers-host-array.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+
+require('../common');
+
+const assert = require('assert');
+const http = require('http');
+
+{
```

Comment:
Nit: this block is not needed?

Reply:
I prefer we keep individual tests in block scopes, even if there's only one test case in the test file. It's not necessary but it keeps everything consistent with test files where they *are* useful, and it makes the test file more easily-extensible and friendly-for-beginners. Need to add a new test case? Hey, put it in some brackets and look at the brackets elsewhere in the test file.

Reply:
In my opinion it only hinders readability when there is only one test case. It's one more unneeded level of indentation. Anyway I don't mind, it is a nit.